.github/CODEOWNERS

```
@@ -104,6 +104,8 @@
 /modules/programs/ne.nix                              @cwyc
 /tests/modules/programs/ne                            @cwyc
 
+/modules/programs/newsboat.nix                        @sumnerevans
```

Can also add the test files:
```suggestion
/modules/programs/newsboat.nix                        @sumnerevans
/tests/modules/programs/newsboat                      @sumnerevans
```